config/dev/spm/sku/BUILD.bazel

```diff
@@ -3,12 +3,123 @@
 # SPDX-License-Identifier: Apache-2.0
 
 load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+load(
+    "//rules:hsm.bzl",
+    "hsm_config_script",
+    "hsm_key_template",
+)
+load(
+    "//rules:hsmtool.bzl",
+    "hsmtool_ecdsa_keygen",
+    "hsmtool_pk11_attrs",
+    "hsmtool_rsa_keygen",
+)
 
-pkg_tar(
+package(default_visibility = ["//visibility:public"])
+
+hsm_key_template(
+    name = "spm-rsa-wrap-v0",
+    export_public_only = True,
+    import_template_public = hsmtool_pk11_attrs({
+        "CKA_ENCRYPT": True,
+        "CKA_VERIFY": True,
+        "CKA_WRAP": True,
+        "CKA_TOKEN": True,
+    }),
+    keygen_params = hsmtool_rsa_keygen(
+        extractable = False,
+        key_length = 3072,
+        label = "spm-rsa-wrap-v0",
+        private_template = {
+            "CKA_CLASS": "CKO_PRIVATE_KEY",
+            "CKA_LABEL": "spm-rsa-unwrap-v0",
+            "CKA_DECRYPT": True,
+            "CKA_SIGN": True,
+            "CKA_TOKEN": True,
+            "CKA_SENSITIVE": True,
+        },
+        public_exponent = 65537,
+        public_template = {
+            "CKA_CLASS": "CKO_PUBLIC_KEY",
+            "CKA_LABEL": "spm-rsa-wrap-v0",
+            "CKA_ENCRYPT": True,
+            "CKA_VERIFY": True,
+            "CKA_TOKEN": True,
+        },
+        wrapping = True,
+    ),
+    type = "rsa",
+)
+
+hsm_key_template(
+    name = "spm-hsm-id-v0",
+    export_public_only = True,
+    import_template_public = hsmtool_pk11_attrs({
+        "CKA_VERIFY": True,
+        "CKA_TOKEN": True,
+    }),
+    # Use NIST P-384 curve for the ECDSA key.
+    keygen_params = hsmtool_ecdsa_keygen(
+        # TODO(moidx): Update to P-384 once the hsmtool supports export ops.
```

Comment:
is this currently a P256 key then? if so maybe update the comment above?

Reply:
Fixed.